main.py

```diff
@@ -0,0 +1,87 @@
+import random
+import os
+from art import logo
+
+# function calculateScore takes in a list parameter and returns sum of list
+def calculateScore(list):
+    count = sum(list)
+
+    if count == 21:
+        return 21
+    elif count > 21:
+        for num in list:
+            if num == 11:
```

Comment:
This is good.  I thought initially that you were always converting 11 to 1 until I saw this condition.